rocky/rocky/scheduler.py

```diff
@@ -309,26 +309,12 @@ def list_tasks(
             raise SchedulerConnectError(extra_message=_("Task list: "))
 
     def get_task_details(self, task_id: str) -> Task:
-        if self.organization_code is None:
-            raise SchedulerTaskNotFound("No organization defined in client.")
-        task_details = Task.model_validate_json(self._get(f"/tasks/{task_id}", "content"))
+        return Task.model_validate_json(self._get(f"/tasks/{task_id}", "content"))
 
-        if task_details.type == "normalizer":
-            organization = task_details.data.raw_data.boefje_meta.organization
-        else:
-            organization = task_details.data.organization
-
-        if organization != self.organization_code:
-            raise SchedulerTaskNotFound()
```

Comment:
These lines where needed because the scheduler has no notion of Which user can access which organization. Also, the scheduler does not return jobs based on their organization, so guessing a valid UUID from a different Organization allowed users to see/restart jobs in organizations that where not theirs. Is there a new mechanism to prevent this?

Reply:
Added back again (partially caused by a merge conflict)